controller/appcontroller.go

```diff
@@ -2459,6 +2467,42 @@ func (ctrl *ApplicationController) newApplicationInformerAndLister() (cache.Shar
 	return informer, lister
 }
 
+func (ctrl *ApplicationController) deleteAppResources(app *appv1.Application) error {
```

Comment:
Maybe add `context.Context` as the first param, and you can pass the `context.TODO` when you call it from `newApplicationInformerAndLister`